homeassistant/components/media_player/kodi.py

```diff
@@ -15,20 +19,49 @@
     SUPPORT_TURN_OFF, MediaPlayerDevice, PLATFORM_SCHEMA)
 from homeassistant.const import (
     STATE_IDLE, STATE_OFF, STATE_PAUSED, STATE_PLAYING, CONF_HOST, CONF_NAME,
-    CONF_PORT, CONF_USERNAME, CONF_PASSWORD)
+    CONF_PORT, CONF_USERNAME, CONF_PASSWORD, EVENT_HOMEASSISTANT_STOP)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['jsonrpc-requests==0.3']
+REQUIREMENTS = ['websockets==3.2']
```

Comment:
@armills excellent, I think the bigger concern was the amount of Kodi protocol that's ending up here. Should we be considering a kodi library?